sidebar.less

```diff
@@ -209,6 +209,11 @@
 		}
 	}
 	
+	.subButtons span:not(.subscribe-button) {	// RES subscribe buttons (+shortcut, +dashboard)
+		width: 80px;
```

Comment:
Setting the non-subscribe buttons to a different size make the sizes inconsistent. It would be better to reduce the size of all the existing selector above (80px looks fine).

This entire selector should be removed.